guides/common/modules/proc_installing-advisor-engine-in-a-disconnected-environment.adoc

```diff
@@ -0,0 +1,27 @@
+[id="installing-advisor-engine-in-a-disconnected-environment"]
+= Installing {advisorengine}
+
+You can pull the {advisorengine} from the Red{nbsp}Hat container registry if your {ProjectServer} has access to the registry.
```

Comment:
What does the user do if the server does not have access to the registry? 

Should registry access be a prerequisite instead?

Reply:
Now I think I understand what's going on. In this procedure, the user must have a connected {ProjectServer}. You might want to make the title clearer. "Installing x" is very generic. Maybe "Installing x on a connected {ProjectServer}. 

I think that what is confusing is that steps 1-2 are required for both this procedure and for import/export procedure. In both cases, the user downloads the image from the registry by using a connected  {ProjectServer}. Personally, I would use a single module with 'ifdefs' for connected and disconnected install, but it's up to you.